src/modules/job-manager/alloc.c

```diff
@@ -66,9 +66,11 @@ static void requeue_pending (struct alloc *alloc, struct job *job)
         job->handle = NULL;
```

Comment:
commit message for "job-manager: support per queue start/stop":

The alloc requests are canceled not the jobs so maybe better would be

> Problem: Individual queues cannot be started/stopped.
> 
> Solution: Update the queue-start RPC to allow individual queues to be started/stopped.
When a queue is stopped, alloc requests for jobs in that queue are canceled, and
the jobs are removed from the alloc queue and prevented from re-entering it.